admin-dev/themes/new-theme/js/pages/order/view/order-prices-refresher.js

```diff
@@ -37,6 +37,8 @@ export default class OrderPricesRefresher {
     $.ajax(this.router.generate('admin_orders_get_prices', {orderId}))
       .then((response) => {
         $(OrderViewPageMap.orderTotal).text(response.orderTotalFormatted);
+        $(OrderViewPageMap.orderDiscountsTotal).text(response.discountsAmountFormatted);
+        $(OrderViewPageMap.orderDiscountsTotal).parent().toggleClass('d-none', !response.discountsAmountDisplayed);
```

Comment:
Could you use something different than a simple `.parent()`, better to target a class instead. Otherwise if we change the template, we're f***ed :sweat_smile:

Reply:
Done :)